lib/web_console/testing/fake_middleware.rb

```diff
@@ -1,11 +1,7 @@
 require 'action_view'
```

Comment:
Yes, we need to keep the line, since the class seems to be not auto loaded by loading `web_console`.